be/src/vec/exec/jni_connector.h

```diff
@@ -165,9 +165,28 @@ class JniConnector {
               _scanner_params(std::move(scanner_params)),
               _column_names(std::move(column_names)) {}
 
+    /**
+    * Use configuration map to provide scan information. The java side should determine how the parameters are parsed.
+    * @param connector_class Java scanner class
+    * @param scanner_params Provided configuration map
+    */
+    JniConnector(std::string connector_class, std::map<std::string, std::string> scanner_params)
+            : _connector_class(std::move(connector_class)),
+              _scanner_params(std::move(scanner_params)) {
+        _is_table_schema = true;
```

Comment:
I should add identifiers on the Java side and decide how to clean up memory on the Java side.

Reply:
Java memory is used here, not direct memory, which is managed by JVM